mirdata/jams_utils.py

```diff
@@ -155,16 +160,28 @@ def jams_converter(
 
     # lyrics
     if lyrics_data is not None:
-        if type(lyrics_data) != list:
+        if not isinstance(lyrics_data, list):
             raise TypeError('lyrics_data should be a list of tuples')
         for lyrics in lyrics_data:
-            if type(lyrics) != tuple:
+            if not isinstance(lyrics, tuple):
                 raise TypeError(
                     'lyrics_data should be a list of tuples, '
                     + 'but contains a {} element'.format(type(lyrics))
                 )
             jam.annotations.append(lyrics_to_jams(lyrics))
 
+    # tags
+    if tags_data is not None:
+        if not isinstance(tags_data, list):
+            raise TypeError('tags_data should be a list of tuples')
+        for tag in tags_data:
+            if not isinstance(tag, tuple):
+                raise TypeError(
+                    'tags_data should be a list of tuples, '
+                    + 'but contains a {} element'.format(type(tag))
+                )
+            jam.annotations.append(tag_gtzan_to_jams(tag))
```

Comment:
`tag_gtzan_to_jams` returns an `Annotation` in the `gtzan` namespace. This `Annotation` will be JAMS-compliant if and only if the `tag` is a GTZAN genre: country, jazz, rock, etc.

This tag converter will not work for other datasets than GTZAN (e.g. MSD, which requires `tag_open`)

Reply:
Yes - this is why I called the method "tag_gtzan_to_jams"

Reply:
I see though - the value is called "tags_data" so it's confusing. I'll update.